### PR TITLE
feat: [sc-23853] Contact Page for NameGuard website

### PR DESCRIPTION
**Pull request by eduramme**
Story details: https://app.shortcut.com/ps-web3/story/23853

### CODE REVIEW

File: apps/nameguard.io/src/app/components/atoms/index.ts [hunk 1]
```diff
@@ -1,3 +1,11 @@
...
 export * from "./CalButton";
 export * from "./ListSectionBadge";
+export * from "./PreSectionText";
+export * from "./SectionText";
+export * from "./ColorfulBg";
+export * from "./SectionTitle";
+export * from "./AirplaneIllustration";
+export * from "./";
+export * from "./";
+export * from "./";
```
Comment (notrab) [inline apps/nameguard.io/src/app/components/atoms/index.ts:11]: Duplicate exports

**Review by lightwalker-eth — CHANGES_REQUESTED** [review]
@eduramme Thanks for these updates :+1: Reviewed and shared feedback.

File: apps/nameguard.io/public/assets/completion_score.svg
[the remark this answers is not in the document]
Comment (eduramme) [reply apps/nameguard.io/public/assets/completion_score.svg:1]: We manage SVGs in React components and HTML differently. For example, HTML SVGs use stroke-dasharray and React uses strokeDasharray in camelCase. I attempted to fix an SVG in a React component and ended up breaking the regular ones. My bad. Thanks for pointing it out :)

**Review by lightwalker-eth — CHANGES_REQUESTED** [review]
@eduramme Thanks for your updates here :+1: Found a few more items in a fresh QA review. Shared feedback for you here: https://app.shortcut.com/ps-web3/story/23853/contact-page-for-nameguard-website#activity-25354

File: package.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
     "eslint": "^8.56.0",
     "prettier": "^3.2.5",
     "rimraf": "^5.0.7",
-    "turbo": "1.13.3"
+    "turbo": "2.0.5"
```
Comment (lightwalker-eth) [inline package.json:19]: This change applies to the whole monorepo and don't suggest we include it in this PR. Can we revert this change?

It would be great if you could create a Shortcut story for the goal of upgrading to turbo 2.x. Please note there's a whole guide of steps that we should follow when we implement this change: https://turbo.build/repo/docs/crafting-your-repository/upgrading

File: apps/nameguard.io/src/app/components/organisms/Header.tsx [hunk 3]
```diff
@@ -61,9 +61,9 @@ export const Header = () => {
             >
               Contact
             </Link>
-            <div className="px-5 py-2 rounded-md bg-black text-white hover:bg-gray-800 transition-all duration-300 whitespace-nowrap">
-              <CalButton>Discuss an integration</CalButton>
-            </div>
+            <CalButton className="px-4 py-2 text-sm rounded-md bg-black text-white hover:bg-gray-800 transition-all duration-300 whitespace-nowrap">
+              Discuss an integration
```
Comment (lightwalker-eth) [inline apps/nameguard.io/src/app/components/organisms/Header.tsx:65]: Can we make this text appear with the same weight as the text of the other buttons / links right next to it? The other  buttons / links right next to it appear to have bold font, while the text in this button doesn't. All these buttons / links should appear bold.

**Review by lightwalker-eth — CHANGES_REQUESTED** [review]
@eduramme Thanks for updates. Shared feedback :+1:

File: package.json (hunk 2 above)
Comment (eduramme) [reply package.json:19]: Seems like we're already using turbo 2.0.5 on main 🤔

Comment (lightwalker-eth) [reply package.json:19]: @eduramme Ah if this change is already on main, then I didn't mean to create confusion with my feedback

Comment (lightwalker-eth) [reply package.json:19]: @eduramme ... but I still understand we need to follow that upgrade process properly that I linked above and I'm not sure we've followed that yet. Therefore please let me know how you suggest we proceed. You're welcome to create a new shortcut story for the turbo upgrade as one option for how we proceed.

**Review by lightwalker-eth — APPROVED** [review]
@eduramme Thanks for the updates. Reviewed and approved :+1: Please take the next steps here to get this merged into main.
